spec/_attachments/ic.did

```diff
@@ -242,4 +259,8 @@ service ic : {
   }) -> (record {canister_id : canister_id});
   provisional_top_up_canister :
     (record { canister_id: canister_id; amount: nat }) -> ();
+
+  // canister loging
+  fetch_logs : (record { canister_id }) -> (fetch_logs_response);
+  fetch_logs_query : (record { canister_id }) -> (fetch_logs_response) query;
```

Comment:
```suggestion
  fetch_logs : (record { canister_id: canister_id }) -> (fetch_logs_response);
  fetch_logs_query : (record { canister_id: canister_id }) -> (fetch_logs_response) query;
```

Reply:
FYI the above is valid candid syntax

Reply:
I see! Then we can leave it as is if you prefer.

Reply:
To keep the same style I switched to `canister_id : canister_id`. It can be prettified later on for all the instances at once.